crates/bench/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# dockerfile for deterministic benchmarking environment
+
+FROM rust:1.72
+
+RUN apt-get update && \
+    apt-get install -y valgrind protobuf-compiler bash && \
+    rm -rf /var/lib/apt/lists/*
+#RUN cargo install --git git@github.com:clockworklabs/iai-callgrind.git --branch main iai-callgrind-runner
+
+ENV CARGO_TARGET_DIR=/projects/SpacetimeDB/linux-target
```

Comment:
This sure looks linux specific. I don't have an issue with that per se as long as it is documented somewhere and, ideally, caught earlier than here with a nice error message, rather than having docker yell at runtime that it can't find these directories.

Reply:
Ah, yes. It was originally non-linux-specific, but docker was incredibly slow when running from a windows FS, so I moved into WSL and haven't looked back. I'll add some docs.

Reply:
Oh, this specific path is INSIDE the docker image.